backend/src/index.ts

```diff
@@ -12,7 +12,7 @@ app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 app.use(cors());
 
-app.use("/", classRoutes);
```

Comment:
can we put this back in for now so frontend doesn't break
app.get('/', (req: Request, res: Response) =>  {
  res.send({ message: "Hello Team Neuron!" });
});